src/cli_parser.cpp

```diff
@@ -19,8 +19,12 @@
 #include <stdexcept>
 #include <string>
 
+// TODO: Write windows/linux specific status codes.
+#ifdef __linux__
 #include <sysexits.h>
-
+#elif _WIN32
+#include <ntstatus.h>
```

Comment:
Why do you include ntstatus.h if you do not use status mapping anyway?

Reply:
It will be implemented in the status codes task. Based on this header.